internal/storage/servers.json

```
@@ -89601,993 +89601,1995 @@
     ]
   },
   "purevpn": {
-    "version": 2,
-    "timestamp": 1622644308,
+    "version": 3,
```

keep the version as `2`, this is just about the server model which is the same. Only the timestamp (auto updated) should be changed, which it is :wink: 

```suggestion
    "version": 2,
```

---

Sorry about that, misunderstood the version number :)
I've fixed it! 

Thank you for all your work!